custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -296,6 +296,10 @@ def district(self):
         return self.nikshay_codes_to_location.get('-'.join(self.patient_detail.PregId.split('-')[:2]))
 
     @property
+    def tu(self):
+        return self.nikshay_codes_to_location.get('-'.join(self.patient_detail.PregId.split('-')[:3]))
```

Comment:
hey, you probably saw my email, but we might need to change this down the line, depending on how we upload the location ids.